packages/matita/matita.0.99.5/opam

```diff
@@ -0,0 +1,54 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "An experimental, interactive theorem prover"
+description:
+  "Matita (that means pencil in italian) is an experimental, interactive theorem prover under development at the Computer Science Department of the University of Bologna."
+maintainer: ["Claudio Sacerdoti Coen"]
+authors: [
+  "HELM Team (Andrea Asperti, Ferruccio Guidi, Luca Padovani, Claudio Sacerdoti Coen, Enrico Tassi, Stefano Zacchiroli et alt.)"
```

Comment:
```suggestion
  "HELM Team"
  "Andrea Asperti"
  "Ferruccio Guidi"
  "Luca Padovani"
  "Claudio Sacerdoti Coen"
  "Enrico Tassi"
  "Stefano Zacchiroli"
```

The format supports multiple authors so it makes sense to make use of it. If you want the "et al." you can maybe use "HELM Team (see <link>)"